source/_docs/guides/front-end-performance.md

```diff
@@ -0,0 +1,481 @@
+  **/
+  add_action( 'send_headers', 'add_header_maxage' );
+  function add_header_maxage() {
+    // some logic that accidentally invalidates full-page cache
+      header('Cache Control: max-age=0');
+  }
+  ```
+   See the [WordPress documentation](https://codex.wordpress.org/Plugin_API/Action_Reference/send_headers){.external} for more details.
+  </div>
+
+  <!-- 2nd pane content -->
+  <div role="tabpanel" class="tab-pane" id="drupal-max-age" markdown="1">
+There could be an existing snippet assumed to disable caching for a single block when in reality it's disabling caching for the entire page:
+
+```php
+/**
+* This is an example of accidentally bypassing cache when rendering a response.
+* Don't use this without proper conditionals wrapped around it.
+**/
+// some logic that accidentally invalidates full-page cache
+$form['#cache'] = ['max-age' => 0];
+```
+ See the [Drupal documentation](https://www.drupal.org/docs/8/api/render-api/cacheability-of-render-arrays){.external} for more details. You can search your custom code for `#cache` to find places where you've interacted with the Cache API.
+  </div>
+</div>
+
+#### Lost or Forgotten Configuration
+Working across many environments in a distributed manner such as Pantheon presents opportunity for configuration to get lost along the way. Using tools like WP-CFM and Drupal 8's Configuration Management System to track configuration alongside code will mitigate these issues, but mistakes do happen. Double-check your site's default caching configurations:
+
+<!-- Nav tabs -->
+<ul class="nav nav-tabs" role="tablist">
+  <!-- Active tab -->
+  <li id="tab-1-id" role="presentation" class="active"><a href="#wp-config-get" aria-controls="wp-config-get" role="tab" data-toggle="tab">WordPress</a></li>
+
+  <!-- 2nd Tab Nav -->
+  <li id="tab-2-id" role="presentation"><a href="#drupal-config-get" aria-controls="drupal-config-get" role="tab" data-toggle="tab">Drupal</a></li>
+</ul>
+
+<!-- Tab panes -->
+<div class="tab-content">
+  <!-- Active pane content -->
+  <div role="tabpanel" class="tab-pane active" id="wp-config-get" markdown="1">
+  The Pantheon Page Cache plugin is already included by our upstream as a Must-Use plugin. Check the plugin settings to make sure you're setting the desired TTL and max-age:
+
+  1. From the WordPress dashboard, click **Settings** > **Pantheon Page Cache**.
+  2. Review the Time to Live, which translates to `max-age`.
+  3. We recommend setting **Default Time to Live (TTL)** to a higher value, like 86400 seconds (one day):
+
+  ![Pantheon Cache Plugin](/source/docs/assets/images/guides/front-end-performance/pantheon-page-cache.png)
+  </div>
+
+  <!-- 2nd pane content -->
+  <div role="tabpanel" class="tab-pane" id="drupal-config-get" markdown="1">
+In Drupal it is very easy to turn off page caching and forget to turn it back on. Use [Terminus](/docs/terminus/) to check and edit caching configurations:
```

Comment:
FYI 

This section describes how to check cache configurations with Terminus because I found that method cool, and I wanted to avoid repeating content from the *Use as Few Requests as Possible* section. Is this okay? Should we change it to the admin ui instead?

thoughts @davidneedham @stevector ?

Reply:
I think the readers are more likely to understand what is happening if we refer to the admin UI instead of a terminus/drush combo.

Reply:
Agreed. I'll get it fixed up. Just to confirm, we're walking through the configs at `/admin/config/development/performance` yeah? Specifically the cache part: 
![screen shot 2017-09-22 at 11 28 25 am](https://user-images.githubusercontent.com/10119525/30754669-3e6ac282-9f89-11e7-8201-f059a149e4e8.png)

What should we recommend they set it to @stevector?

Reply:
Yes, that's it. It's hard to make one standard recommendation here. The Drupal 8 default is 10 minutes. But if Pantheon Advanced Page Cache is being used well, then you could go much, much higher (hours, days, weeks). But I don't think we have enough guidance written yet on validating Pantheon Advanced Page Cache is being used well on a site to make a blanket recommendation of some high number.

How about something like

> The Drupal 8 default setting is 10 minutes. You can set much higher cache max ages when using the Pantheon Advanced Page Cache Module to clear specific pages when your underlying data is updated.